googlemediaframework/src/main/java/com/google/android/libraries/mediaframework/exoplayerextensions/ExoplayerWrapper.java

```diff
@@ -74,17 +77,14 @@
     /**
      * Invoked with the results from a {@link RendererBuilder}.
      *
-     * @param trackNames The names of the available tracks, indexed by {@link ExoplayerWrapper}
-     *                   TYPE_* constants. May be null if the track names are unknown. An individual
-     *                   element may be null if the track names are unknown for the corresponding
-     *                   type.
+     * @param trackNames The names of the available tracks, indexed by {@link ExoplayerWrapper} TYPE_*
```

Comment:
Please revert these comment reformat changes (unless there's some change in the text, we don't want to submit format changes in CLs with logic changes)